Library/Formula/svmrank.rb

```diff
@@ -0,0 +1,16 @@
+class Svmrank < Formula
+  homepage "http://www.cs.cornell.edu/people/tj/svm_light/svm_rank.html"
+  url "http://download.joachims.org/svm_rank/current/svm_rank.tar.gz"
+  version "1.00"
+  sha1 "0f81e563aed064fbca0477d8f8a1d640b1b9c2f9"
+
+  def install
+    system "make all"
+    bin.install "svm_rank_classify"
+    bin.install "svm_rank_learn"
+  end
+
+  test do
+    system "false"
```

Comment:
Can this test be modified to do something more substantial than e.g. `--version` or `--help`? See `cmake.rb` for an example of an application formula with a good test and `tinyxml2.rb` for an example of a library formula with a good test. Thanks!